daliuge-engine/dlg/manager/proc_daemon.py

```diff
@@ -298,6 +336,19 @@ def _callback(
             logger.info("Zeroconf started")
             return
 
+    def startTM(self):
+        tool = get_tool()
+        args = ["--host", "0.0.0.0"]
+        args += self._verbosity_as_cmdline()
+
+        ## Translator does not support daemonising yet.
+        # if self._daemonise:
+        #     args += ['-d']
+        logger.info("Starting Translator Manager with args: %s", (" ".join(args)))
```

Comment:
Technically, this means daliuge-engine/setup.py should have `daliuge-translator` as a dependency.